bin/zerobufCxx.py

```diff
@@ -822,6 +903,8 @@ def is_dynamic(self, attrib, fbsFile):
 
     def parse_members(self, fbsFile):
         dynamic_type_index = 0
+        self.json_schema['properties'] = {}
+        properties = self.json_schema['properties']
         for attrib in self.attributes:
```

Comment:
I'd like to suggest a big change in this for loop, feel free to do it, but I think it will improve readability. Actually it's two different proposals, if you don't want to do it any of them at least you should add more comments because the code it's quite convoluted.
Here I understand that you are doing two independent things in the same loop and it gets big. For readability I think it's better to separate the two processings. The easiest way to do it is to have two separate loops (despite the if sentences are going to be the same), one for the original code and another for the new stuff about properties. For maintainability the loop code should be refactored, and the only way I can imagine is using something similar to a visitor factor. In python you do it without to much hassle. Create a function for doing the loop and the if parts, this function takes as an argument a list of objects with callbacks to deal with each specific case. Declare two classes, one with the callbacks for the original code and another for the properties. With proper function names you won't need to write any single comment.